lib/openstudio-standards/standards/Standards.CoilHeatingDXSingleSpeed.rb

```diff
@@ -129,7 +129,7 @@ def coil_heating_dx_single_speed_standard_minimum_cop(coil_heating_dx_single_spe
     end
 
     # If PTHP, use equations
-    if sub_category == 'PTHP'
+    if sub_category == 'PTHP' && !ac_props['pthp_cop_coefficient_1'].nil? && !ac_props['pthp_cop_coefficient_2'].nil
```

Comment:
90.1-2019 and 90.1-2016 still use the coefficients, just not for units less than 7 kBtu/h and for the larger ones that have a capacity greater than 15 kBtu/h. An EER should be used in these cases:

![image](https://user-images.githubusercontent.com/5149279/234357960-b315d1bf-45ec-431e-866a-a2131d79bbcb.png)

![image](https://user-images.githubusercontent.com/5149279/234358038-1390322c-9882-4adb-885d-5eab4e526d3a.png)

Will this change prevent the EER values to be assigned if the `sub_category` is `PTHP`?

The same thing should be happening for PTAC.

Reply:
All this method does is ensure that the PTHP coefficients are valid before applying them.  If other efficiency values are defined, those get applied instead a few lines later on in the method.